HOW_TO_USE_STRAPI_GUIDE.md

```diff
@@ -0,0 +1,92 @@
+# How to use strapi
+
+# Requirements
+1. Make sure you have followed the [contributor guide](CONTRIBUTING.md) to setup the development environment
+2. Get ready for a decent amount of reading
+
+# Setup
+- Once you've got the development environment setup, then you will be able to
+  access the strapi admin dashboard running on http://localhost:1337/admin.
+- You'll have to create an admin account to be able to use the strapi dashboard, so
+  enter a username and password that you'll remember. It should look like this.
+  ![Admin Dashboard](./images/how-to-use-strapi/admin.png)
+- Once you've created an account, we can get into the fun stuff!
+
+# Creating a test user
+- To be able to test authenticated routes in the api, you'll need to create a test user
+- To create a test user head to the users collection ![Users Collection](./images/how-to-use-strapi/users_collection.png)
+- Once on that page, click on "+ Add New Users" on the top right of the screen ![Add New Users Button](./images/how-to-use-strapi/add_new_user.png)
+- Once on that page, fill out all these fields since they're required ![Users Fields][./images/how-to-use-strapi/users_fields.png]
+- After that, click the green save button on the top right
+
+# Testing the api
+- I'm going to be using insomnia in this tutorial, but you can use any other REST client
+- If any of the endpoints you want to test are public, then all you need to do is make the request for it
+  normally since it doesn't require any authentication
+- If one of your endpoints is protected by authentication, and you try to make a request to it without signing in as your test user,
+  you'll get a 403 forbidden error ![Forbidden](./images/how-to-use-strapi/forbidden.png).
+- To sign in as your test user, make sure to make a POST request to https://localhost:1337/auth/local
+  with a JSON object with the keys identifier and password then click the purple send button. Identifier is just your username or email
```

Comment:
What is keys identifier?

Reply:
I explain later in the paragraph “identifier is just your username and email. I also provide an image as an example. It just doesn’t render in here.